test/utils/fixtures.ts

```diff
@@ -329,6 +334,10 @@ export const CLASSIC_QUOTE_EXACT_IN_LARGE = createClassicQuote(
   { quote: AMOUNT_LARGE, quoteGasAdjusted: AMOUNT_LARGE_GAS_ADJUSTED },
   { type: 'EXACT_INPUT' }
 );
+export const CLASSIC_QUOTE_EXACT_IN_LARGE_PORTION = createClassicQuote(
```

Comment:
nit - you can inline `portionBips: PORTION_BIPS, portionRecipient: PORTION_RECIPIENT` into `CLASSIC_QUOTE_EXACT_IN_LARGE`